history.py

```diff
@@ -0,0 +1,23 @@
+from catalog import models as cmod
+
+
+class LastFiveMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+
+        id_list = request.session.get('lastfive', [])
+
+        request.last_five = []
+        for stuff in id_list:
```

Comment:
'stuff' isn't a super great variable name. try something a little more specific/pertinent

Reply:
Stuff it.